jplag/src/main/java/de/jplag/Submission.java

```diff
@@ -50,6 +50,11 @@
      */
     private TokenList tokenList;
 
+    /**
+     * baseCodeMatch
+     */
+    private JPlagComparison baseCodeMatch;
+
```

Comment:
The name and description are a bit misleading as comparisons are no matches. A comparison can have multiple matches. In some places in our source code, this still might be wrong. I would propose using `baseCodeComparison ` here. Also, the JDoc comment is a bit redundant, as it gives no valuable information.